### DISCUSSION TITLE
How to access the web interface of a container from outside the network

### DISCUSSION THREAD

**Discussion by flodes80** [bubuntux/nordlynx#93] [Q&A]
Hello, 

I have successfully set up the vpn for my "Transmission" container. However I can only access the web interface from my local network. As soon as I want to access it from outside (so not from my home) I cannot access it. I'm quite new to network management, I must have forgotten something. I guess my vpn container does not allow access outside the local network.

Thanks in advance!

**Comment by instantdreams** [comment]
There are three options here.
1. Forward your Transmission web ui port (default 9091) through your router
2. Implement a reverse proxy on your home network and use a vanity domain so you can forward "transmission.vanitydomain.org" to your transmission service
3. Set up a wireguard or other VPN on your home network and connect to it, then access your Transmission container as if local

Of these, the third is the most secure, the first is the least secure.
